src/main/sensors/current.c

```diff
@@ -107,9 +107,10 @@ static int32_t currentMeterADCToCentiamps(const uint16_t src)
     const currentSensorADCConfig_t *config = currentSensorADCConfig();
 
     int32_t millivolts = ((uint32_t)src * ADCVREF) / 4096;
-    millivolts -= config->offset;
+	//y=mx+b m is scale in (A/mV) and b is offset in (A)
+	float amps = (millivolts / (float)config->scale + (float)config->offset;
```

Comment:
There's one too many opening brackets on this line. Making sure that your change builds at least with the target you are testing with **before** opening a pull request is a good way to eliminate this kind of problem.